### ISSUE TITLE
MAMP Pro Hosts - white screen

### ISSUE THREAD

**Issue by maxxscho** [bdkjones/CodeKit2#23]
I am working with MAMP Pro and use "Hosts" for my projects, but I have problems running the built-in server. I've set up the External server address (http://my-host.dev) and the external server port (80).
Sometimes I got the "missing index file"-screen but mostly I got only a white screen.

Btw. is there a way to auto-reload browsers like in CK 1.x?

Edit: I'm working on a CodeIgniter Project - so I have an index.php.

**Comment by mdavis1982** [comment]
@maxxscho CodeKit should automatically reload browser as that's one of its primary features. What are you experiencing?

**Comment by maxxscho** [comment]
I've set up the built-in server as described in Bryan's video and if I hit the "Preview" button, I only see a white screen with no content.
I have tried with different virtual hosts on MAMP Pro and also with the default 'localhost', but it doesn't work. I mean it's a great feature with the server right in Codekit 2 but I also liked the auto-reload feature in CodeKit 1.x where I choosed the browser and the tabs for auto-reload. It was easier to set up.
